packages/jupytercad/src/_interface/box.d.ts

```diff
@@ -0,0 +1,41 @@
+/* tslint:disable */
```

Comment:
This entire `_interface` directory is not to be committed and should be in the `.gitignore`